paddlenlp/transformers/xlnet/tokenizer.py

```diff
@@ -251,14 +261,14 @@ def convert_ids_to_tokens(self, ids, skip_special_tokens=False):
         a sequence of tokens, using the vocabulary and added tokens.
 
         Args:
-            ids (`int` or `List[int]`):
+            ids (int or List[int]):
                 The token id (or token ids) to be converted to token(s).
-            skip_special_tokens (`bool`, optional):
+            skip_special_tokens (bool, optional):
                 Whether or not to remove special tokens in the decoding.
-                Defaults to ``False`` and we do not remove special tokens.
+                Defaults to `False` and we do not remove special tokens.
 
         Returns:
-            `str` or `List[str]`: The decoded token(s).
+            str or List[str]: The decoded token(s).
         """
         if not isinstance(ids, (list, tuple)):
```

Comment:
Is a tuple output also allowed in "convert_ids_to_tokens"?

Reply:
nope :(